bin/templatemap_generator.php

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
```

Comment:
> I think @Ocramius comment is about placing license header, not about where template map generator

No, I actually am against having the template map generator in the skeleton.

First of all, it could live in a separate module, like all other MVC-ish modules in here.
Second, I already built something that completely removes the need for a template map, and is compatible with v3 - https://github.com/Ocramius/OcraCachedViewResolver

This, in my opinion, is micro-optimization and complication.